tokens/properties/components/LookupMenu.json

```diff
@@ -0,0 +1,91 @@
+{
+	"wikit-LookupMenu": {
+		"background-color": {
+			"value": "{background.color.base.default.value}"
+		},
+		"border" : {
+			"style" : {
+				"value": "{border.style.base.value}"
+			},
+			"width" : {
+				"value": "{border.width.thin.value}"
+			},
+			"radius" : {
+				"value": "{border.style.base.value}"
+			},
+			"color" : {
+				"value": "{border.color.base.default.value}"
+			}
+		},
+		"box-shadow": {
+			"value": "{box-shadow.elevation.menu.value}"
+		},
+		"item" : {
+			"padding-vertical": {
```

Comment:
This should probably be `spaceing-vertical` or something similar

Reply:
they are referenced as padding-vertical and padding-horizontal in the design files. don't know what to make of this.

Reply:
No, I don't think that is has to be the same as the base token, either.
Rather that it should not needlessly preclude the specific implementation, and apparently there is some work going on removing some implementation detail from earlier token names. OTOH, if that term (i.e. `padding`) is a good way to talk about it from a design perspective, then it's probably OK.

Reply:
I asked @SaiSan-WMDE and we thought that since this is a conceptual issue, we can probably leave it like this and discuss this topic with the design team and change it retroactively when a consensus is achieved.